target/linux/ar71xx/image/tp-link.mk

```diff
@@ -574,7 +574,16 @@ define Device/tl-wa901nd-v4
   TPLINK_HWID := 0x09010004
   IMAGE/factory.bin := append-rootfs | mktplinkfw factory -C EU
 endef
-TARGET_DEVICES += tl-wa901nd-v1 tl-wa901nd-v2 tl-wa901nd-v3 tl-wa901nd-v4
+
+define Device/tl-wa901nd-v5
```

Comment:
Could you please simplify this definition (we both don't like code duplication)?
You can do something like:

```
define Device/tl-wa901nd-v5
  $(Device/tl-wa901nd-v4)
  ...here overwrite only different values...
endef
```

Cheers,
Piotr